### PR TITLE
Update view-only-meeting-experience.md

### PR DESCRIPTION
**Pull request by chbalaki**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/view-only-meeting-experience.md [hunk 1]
```diff
@@ -34,6 +31,9 @@ Attendees who join after the main meeting capacity has been reached will have a
 
 Attendees will be able to join the view-only experience through desktop, web, and Teams mobile (Android and iOS).
 
+> [!Note]
+> View-only broadcasts is available in Microsoft 365 E3/E5 and Microsoft 365 A3/A5. This feature will be enabled March 1, 2021 as default OFF. The feature in Microsoft 365 Government Community Cloud (GCC) will begin to roll out at the end of March 2021. Government Community Cloud High (GCCH) and Department of Defense (DoD) will roll out at a later date. You must change the default policy after that date if you want to have the feature be default ON. Use PowerShell to enable the policy `Set-CsTeamsMeetingPolicy -Identity Global -StreamingAttendeeMode Enabled`.
```
Comment (JohanFreelancer9) [inline Teams/view-only-meeting-experience.md:35]:
```suggestion
> View-only broadcasts are available in Microsoft 365 E3/E5 and Microsoft 365 A3/A5. This feature will be enabled on March 1, 2021 as default OFF. The feature in Microsoft 365 Government Community Cloud (GCC) will begin to roll out at the end of March 2021. Government Community Cloud High (GCCH) and Department of Defense (DoD) will roll out at a later date. You must change the default policy after that date if you want to have the feature default to be ON. Use PowerShell to enable the policy `Set-CsTeamsMeetingPolicy -Identity Global -StreamingAttendeeMode Enabled`.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chbalaki, kindly commit the suggested changes. Many thanks.

**Comment by chbalaki** [comment]
Thanks @JohanFreelancer9 - I spoke with some other PMs and we are going to not go forward with my proposed changes right now. Thank you however for the PR review!

**Pull request closed without merging**